### PR TITLE
Add compare table report.

### PR DESCRIPTION
**Pull request by crazycs520**
* Add compare table report
* Add memory usage in node_load table.
* fix bug of AvgMaxMinTableDef.

Test

```shell
cd pkg/apiserver/diagnose
go test -check.f TestCompareTable > out
```

```
node
process cpu usage
DIFF_RATIO = max( (t2.AVG-t1.AVG)/max(t2.AVG, t1.AVG) , (t2.MAX-t1.MAX)/max(t2.MAX, t1.MAX) , (t2.MIN-t1.MIN)/max(t2.MIN, t1.MIN) )

instance             |    job                    |    t1.AVG        |    t1.MAX        |    t1.MIN        |    DIFF_RATIO    |    t2.AVG        |    t2.MAX        |    t2.MIN    
172.16.5.40:23151    |    tikv                   |    3.27          |    4.37          |    2.58          |    0.33          |    4.29          |    4.53          |    3.84      
172.16.5.40:20151    |    tikv                   |    2.99          |    3.42          |    2.18          |    0.31          |    4.14          |    4.99          |    2.79      
172.16.5.40:9099     |    overwritten-cluster    |    0.0008        |    0.001         |    0.0007        |    0.2           |    0.001         |    0.001         |    0.0007    
172.16.5.40:19110    |    overwritten-nodes      |    0.01          |    0.02          |    0.01          |    0             |    0.01          |    0.02          |    0.01      
172.16.5.40:10089    |    tidb                   |    15.68         |    15.86         |    15.38         |    -0.07         |    14.99         |    15.43         |    14.38     
172.16.5.40:24799    |    pd                     |    0.55          |    0.56          |    0.55          |    -0.11         |    0.51          |    0.53          |    0.49      
172.16.5.40:21151    |    tikv                   |    4.25          |    4.55          |    3.78          |    -0.24         |    3.53          |    4.78          |    2.87      
```

### CODE REVIEW

File: pkg/apiserver/diagnose/compare.go [hunk 1]
```diff
@@ -0,0 +1,491 @@
...
+		f2, err := parseFloat(row2[idx])
+		if err != nil {
+			return 0, err
+		}
+		if f1 == f2 {
+			continue
+		}
+		ratio := (f2 - f1) / math.Max(f1, f2)
+		if math.Abs(ratio) > math.Abs(maxRatio) {
+			maxRatio = ratio
```
Comment (reafans) [inline pkg/apiserver/diagnose/compare.go]: How about add the max diff name in  `maxRatio` .
For example , if the max diff is avg(value), then join it to maxRatio as one string.

Comment (crazycs520) [reply pkg/apiserver/diagnose/compare.go]: image

After trying, I prefer the original format...

File: pkg/apiserver/diagnose/report_test.go [hunk 2]
```diff
@@ -91,6 +191,9 @@ func printRows(t *TableDef) {
 		return
 	}
 
+	fmt.Println(strings.Join(t.Category, " - "))
+	fmt.Println(t.Title)
+	fmt.Println(t.CommentEN)
```
Comment (djshow832) [inline pkg/apiserver/diagnose/report_test.go]: Is this necessary?

Comment (crazycs520) [reply pkg/apiserver/diagnose/report_test.go]: It is just used in the test. Move to the front will know which table has 0 rows.

**Pull request merged**